src/mmw/js/src/core/modals/views.js

```diff
@@ -19,6 +19,7 @@ var _ = require('lodash'),
     modalIframeTmpl = require('./templates/iframeModal.html'),
     vizerUrls = require('../settings').get('vizer_urls'),
 
+    GWLFE = 'gwlfe',
```

Comment:
Can we import this from `modeling/models`? I'd prefer not defining the same constant in two places.

Reply:
This is in `core` which is usually agnostic to the other directories (which import it). I think instead it's time to move the model constants out into `core/utils`. We reference them in analyze, modeling, and now here.